src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -405,7 +402,7 @@ void cannotCreateTitleLevelRequestForUnknownInstance(String tlrFeatureStatus,
 
     Response postResponse = requestsClient.attemptCreate(new RequestBuilder()
       .withRequestType(requestType)
-      .withRequestLevel(requestLevel)
+      .withRequestLevel("Item")
```

Comment:
There are methods titleRequestLevel() and itemRequestLevel() in this builder

Reply:
replaced